src/lib.rs

```diff
@@ -2168,7 +2195,7 @@ impl<'a, L: 'static + Ledger, Backend: 'a + WalletBackend<'a, L> + Send + Sync>
     /// Compute the balance frozen records of the given asset type owned by the given address.
     pub async fn frozen_balance(&self, account: &UserAddress, asset: &AssetCode) -> u64 {
```

Comment:
Can we change this to `frozen_balance_breakdown` to be more consistent? Don't have to add `frozen_balance(self, asset)` now, but we should at least reserve the name for it.